jackbox_patcher/assets/l10n/app_de.arb

```diff
@@ -237,4 +237,4 @@
     "small_description":"Beschreibung",
 
     "patreon_subscribers":"Patreon-Supporter"
-}
\ No newline at end of file
+}
```

Comment:
Idk why this is a sudden change now, because it wasn't before. So maybe someone forgot to close the bracket.